src/plans/D3MAavePlan.sol

```diff
@@ -25,35 +25,35 @@ interface TokenLike {
 
 interface LendingPoolLike {
     function getReserveData(address asset) external view returns (
-        uint256,    // configuration
-        uint128,    // the liquidity index. Expressed in ray
-        uint128,    // variable borrow index. Expressed in ray
-        uint128,    // the current supply rate. Expressed in ray
-        uint128,    // the current variable borrow rate. Expressed in ray
-        uint128,    // the current stable borrow rate. Expressed in ray
-        uint40,     // last updated timestamp
-        address,    // address of the adai interest bearing token
-        address,    // address of the stable debt token
-        address,    // address of the variable debt token
-        address,    // address of the interest rate strategy
-        uint8       // the id of the reserve
+        uint256, // configuration
+        uint128, // the liquidity index. Expressed in ray
+        uint128, // variable borrow index. Expressed in ray
+        uint128, // the current supply rate. Expressed in ray
+        uint128, // the current variable borrow rate. Expressed in ray
+        uint128, // the current stable borrow rate. Expressed in ray
+        uint40,  // last updated timestamp
+        address, // address of the adai interest bearing token
+        address, // address of the stable debt token
+        address, // address of the variable debt token
+        address, // address of the interest rate strategy
+        uint8    // the id of the reserve
     );
 }
 
 interface InterestRateStrategyLike {
     function OPTIMAL_UTILIZATION_RATE() external view returns (uint256);
-    function EXCESS_UTILIZATION_RATE() external view returns (uint256);
-    function variableRateSlope1() external view returns (uint256);
-    function variableRateSlope2() external view returns (uint256);
-    function baseVariableBorrowRate() external view returns (uint256);
+    function EXCESS_UTILIZATION_RATE()  external view returns (uint256);
```

Comment:
missed this interface

Reply:
this is on purpose, don't you think it looks much better aligned?
the alignment is not a standard across maker repos anyway, it's jsut something that should be the same across pools and plans for easy diff.

Reply:
I'm not sure about this, maybe it could be discussed more in a separate PR as it breaks from previous style patterns

Reply:
reverted that, too tired to argue on this.

Reply:
Interfaces are supposed to be consumed across the space and not contract specific so I would keep the code style consistent